docs/ReleaseNotes.md

```diff
@@ -1,6 +1,32 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 3.1.2
+Acuminator 3.1.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 3.1.2, the following enhancements have been implemented:
+ - The [PX1096](diagnostics/PX1096.md) diagnostic has been added to check signature of methods with the `PXOverride` attribute.
+ - The [PX1007](diagnostics/PX1007.md) diagnostic no longer checks the following DAC fields for a code annotation:
+   - `Selected`
+   - `Attributes`
+   - `GroupMask`
+   - `CompanyID`
+   - `CompanyMask`
+   - `DeletedDatabaseRecord`
+ - The [PX1007](diagnostics/PX1007.md) diagnostic now checks that mapped DAC fields of projection DACs have the `inheritoc` XML tag
```

Comment:
That's not complete description + there is a mistype in the tag name. 
PX1007 now checks that mapped DAC field **properties** have `inheritdoc` tag with `cref` attribute pointing to the mapped DAC field property of the original DAC. An example of correctly declared projection DAC property will be really helpful here and a link to the inheritdoc tag documentation:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc

Reply:
Is it okay if I give the link to inheritdoc documnetation and details about the original mapped property in the article about PX1007? This is too specific info for RNs

Reply:
Yes, it's okay.